internal/services/signalr/registration.go

```diff
@@ -21,7 +21,8 @@ func (r Registration) WebsiteCategories() []string {
 // SupportedDataSources returns the supported Data Sources supported by this Service
 func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
 	return map[string]*pluginsdk.Resource{
-		"azurerm_signalr_service": dataSourceArmSignalRService(),
+		"azurerm_signalr_service":    dataSourceArmSignalRService(),
+		"azurerm_signalr_web_pubsub": dataSourceSignalrWebPubsub(),
```

Comment:
this can remain `azurerm_web_pubsub` here fwiw:

```suggestion
		"azurerm_web_pubsub": dataSourceSignalrWebPubsub(),
```